### DISCUSSION TITLE
How to enable team collaboration with Budibase App

### DISCUSSION THREAD

**Discussion by ayesha-siddiqa20** [Budibase/budibase#8384] [Ideas]
When one person is modifying a Budibase App, the app will be locked and other team members (who are given permission) cannot access the app until the previous user unlocks it. This brings an issue to working simultaneously. Is there a way to fix this issue or work around it?

Thanks.

**Comment by melohagan** [comment]
Hey @ayesha-siddiqa20 

Currently there is no way to get around this, but it is a good idea! 

Revision history and version control is currently being worked on this quarter. Extending team collaboration could potentially be a nice add on after that work has finished. 
https://github.com/Budibase/budibase/projects/10
